crates/core/src/factories/core_fetch/test.rs

```diff
@@ -33,8 +35,8 @@ mod tests {
             std::sync::atomic::AtomicU64::new(0);
 
         #[derive(Debug)]
-        struct R;
-        impl Report for R {
+        struct MockReport;
```

Comment:
I found the code very hard to read with single letters so I also did some renaming while fixing the test. I hope that's okay.